README.md

```diff
@@ -8,6 +8,7 @@ To add a conference to the list, please [contribute](#contributing)!
 
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
+| [IN/Clojure](http://inclojure.org/) | 12-13 Jan | Bengaluru | IN/Clojure is India's annual Clojure conference. It is also the only Clojure conference in all of Asia.
```

Comment:
Thanks for adding this. Also, thank you for sticking to the PR template!!

This list is in reverse chronological order, so the conferences occurring first will need to be added in the bottom of the list. Please add this in the bottom and then lgtm.

Reply:
My bad